packages/imperative/src/config/src/api/ConfigLayers.ts

```diff
@@ -31,7 +33,7 @@ export class ConfigLayers extends ConfigApi {
      * @param opts The user and global flags that indicate which of the four
      *             config files (aka layers) is to be read.
      */
-    public read(opts?: { user: boolean; global: boolean }) {
+    public read(opts?: { user: boolean; global: boolean; ignoreErrors: boolean}) {
```

Comment:
This may be out of scope, but should we use either an existing interface (or create a new one)?

If creating a new one... should we simplify the IConfigLayer and extend from the one we create here? 😋

Reply:
i instead modified packages\imperative\src\config\src\Config.ts to set ignoreErrors to false if theres no existing ignoreErrors. now that im thinking about that retroactively, we are always going to have ignoreErrors set now and im not sure if that's wanted behavior